src/cljs/main/broadfcui/page/workspace/summary/tab.cljs

```diff
@@ -175,79 +177,91 @@
-             [:div {:style {:fontSize "80%"}} (str "Note: the billing account associated with " (:namespace workspace-id) " will be charged.")]])))
-      (render-detail-box
-        4
-        "Analysis Submissions"
-        (style/create-paragraph
-          (let [count-all (apply + (vals submissions-count))]
-            [:div {}
-             (str count-all " Submission" (when-not (= 1 count-all) "s"))
-             (when (pos? count-all)
-               [:ul {:style {:marginTop 0}}
-                (for [[status subs] (sort submissions-count)]
-                  [:li {} (str subs " " status)])])])))
+       "Workspace Access"
+
+       "Access Level"
+       (style/prettify-access-level user-access-level)
+
+       (str "Workspace Owner" (when (> (count owners) 1) "s"))
+       (interpose ", " owners)
+
+       "Created By"
+       [:div {}
+        [:div {} createdBy]
+        [:div {} (common/format-date createdDate)]])
+
       (render-detail-box
-        5
-        "Tags"
-        (style/create-paragraph
-          (cond editing? [comps/TagAutocomplete {:tags processed-tags :ref "tags-autocomplete"}]
-                (empty? processed-tags) [:em {} "No tags provided"]
-                :else [:div {}
-                       (for [tag processed-tags]
-                         [:div {:style {:display "inline-block" :background (:tag-background style/colors)
-                                        :color (:tag-foreground style/colors) :margin "0.1rem 0.1rem"
-                                        :borderRadius 3 :padding "0.2rem 0.5rem"}} tag])])))]
+       "Storage & Analysis"
+
+       "Google Bucket"
+       [:div {}
+        (case bucket-access?
+          nil [:div {:style {:position "absolute" :marginTop "-1.5em"}}
+               [comps/Spinner {:height "1.5ex"}]]
+          true (style/create-link {:text bucketName
+                                   :href (str moncommon/google-cloud-context bucketName "/")
+                                   :style {:color "-webkit-link" :textDecoration "underline"}
+                                   :title "Click to open the Google Cloud Storage browser for this bucket"
+                                   :target "_blank"})
+          false bucketName)
+        (when (not reader?)
+          [:div {:style {:lineHeight "initial"}}
+           [:div {} (str "Total Estimated Storage Fee per month = " storage-cost)]
+           [:div {:style {:fontSize "80%"}} (str "Note: the billing account associated with " (:namespace workspace-id) " will be charged.")]])]
+
+       "Analysis Submissions"
+       (let [count-all (apply + (vals submissions-count))]
+         [:div {}
+          (str count-all " Submission" (when-not (= 1 count-all) "s"))
+          (when (pos? count-all)
+            [:ul {:style {:marginTop 0}}
+             (for [[status subs] (sort submissions-count)]
+               [:li {} (str subs " " status)])])]))]
```

Comment:
Using map, this would be:
```cljs
(map (fn [[status subs]] [:li {} (str subs " " status)]) (sort submissions-count))
```
The `for` is clearer in my opinion. I wish we used it more often.